lib/workers/repository/update/pr/changelog/github.spec.ts

```diff
@@ -356,15 +358,13 @@ describe('workers/repository/update/pr/changelog/github', () => {
     });
 
     it('works with same version releases but different prefix', async () => {
-      httpMock
-        .scope('https://api.github.com/')
-        .get('/repos/chalk/chalk/tags?per_page=100')
-        .reply(200, [
-          { name: 'v1.0.1' },
-          { name: '1.0.1' },
-          { name: 'correctPrefix/target@1.0.1' },
-          { name: 'wrongPrefix/target-1.0.1' },
-        ]);
+      const githubTagsMock = jest.spyOn(githubGraphql, 'queryTags');
+      githubTagsMock.mockResolvedValue([
+        { version: 'v1.0.1' },
+        { version: '1.0.1' },
+        { version: 'correctPrefix/target@1.0.1' },
+        { version: 'wrongPrefix/target-1.0.1' },
+      ] as Partial<GithubTagItem[]> as never);
```

Comment:
i mean our `partial<T>` test util function 😅

Reply:
Ah, okay, this one is neat 👍